popple.go

```diff
@@ -19,6 +19,12 @@ const DEFAULT_WORKERS uint = 4
 const DEFAULT_JOBS uint = 128
 
 func main() {
+
+	if len(os.Args) >= 2 && (os.Args[1] == "version") {
+		fmt.Printf("Popple version : %s", version)
+		os.Exit(0)
+	}
+
```

Comment:
I appreciate where you're going with this, but Popple isn't an interactive command line tool so I don't think we should add a switch like this. Please remove this.